cls/SourceControl/Git/WebUIDriver.cls

```diff
@@ -160,13 +160,17 @@ ClassMethod Uncommitted() As %SystemBase
         quit:key=""
         // Check that current user has files(key) uncommitted and only %Push if they do
         set filename = ##class(Utils).FullExternalName(key)
-        set sc=##class(SourceControl.Git.Change).GetUncommitted(filename,.tAction,.tInternalName,.UncommittedUser,.tSource,.UncommittedLastUpdated)
-        if $$$ISOK(sc) && ($data(tAction)&&(UncommittedUser=$username)) {
+        if (($ISVALIDNUM(key)) && (files(key) '= "")){
```

Comment:
Why do we need to check ISVALIDNUM?

Reply:
In `Utils.GitStatus()`, if we want to include all files, then for the files that don't have an internal name but are still tracked by git, I use `$increment` to generate keys. For files that have internal names, the internal names are the keys and the filepath is the value associated with the key. 

So in `WebUIDriver.Uncommitted()`, if the key is a number and the value is not empty, then it indicates that the file is tracked by Git but not by the Uncommitted queue. Therefore, any user should be able to take action on those changes.